sdks/python/apache_beam/io/avroio_test.py

```diff
@@ -157,7 +158,9 @@ def test_source_display_data(self):
     # No extra avro parameters for AvroSource.
     expected_items = [
         DisplayDataItemMatcher('compression', 'auto'),
-        DisplayDataItemMatcher('file_pattern', file_name)]
+        DisplayDataItemMatcher(
+            'file_pattern',
+            'StaticValueProvider(type=str, value=\'%s\')' % file_name)]
```

Comment:
By now, I'm leaving the current format even for StaticValueProvider given that more options (not just strings can potentially be parameterized). In any case, the change is minimal indeed.